frankenphp.go

```diff
@@ -501,6 +501,80 @@ func ServeHTTP(responseWriter http.ResponseWriter, request *http.Request) error
 	return nil
 }
 
+//export go_putenv
+func go_putenv(str *C.char, length C.int) C.bool {
+	// Create a byte slice from C string with a specified length
+	s := C.GoBytes(unsafe.Pointer(str), length)
+
+	// Convert byte slice to string
+	envString := string(s)
+
+	// Check if '=' is present in the string
+	if idx := strings.IndexByte(envString, '='); idx != -1 {
+		// '=' found, set the environment variable
+		key := envString[:idx]
+		val := envString[idx+1:]
```

Comment:
Isn't the out-of-range problem in the case of `FOO=` still there?

We should use `strings.Cut()` here too, it's the easiest way.

Reply:
always forget that exists...